network/links.md

```diff
@@ -77,28 +84,33 @@ default the `-p` flag binds the specified port to all interfaces on
 the host machine. But you can also specify a binding to a specific
 interface, for example only to the `localhost`.
 
-    $ docker run -d -p 127.0.0.1:80:5000 training/webapp python app.py
+```console
+$ docker run -d -p 127.0.0.1:80:5000 training/webapp python app.py
 
 This would bind port 5000 inside the container to port 80 on the
 `localhost` or `127.0.0.1` interface on the host machine.
 
 Or, to bind port 5000 of the container to a dynamic port but only on the
 `localhost`, you could use:
 
-    $ docker run -d -p 127.0.0.1::5000 training/webapp python app.py
+```console
+$ docker run -d -p 127.0.0.1::5000 training/webapp python app.py
 
 You can also bind UDP and SCTP (typically used by telecom protocols such as SIGTRAN, Diameter, and S1AP/X2AP) ports by adding a trailing `/udp` or `/sctp`. For example:
 
-    $ docker run -d -p 127.0.0.1:80:5000/udp training/webapp python app.py
+```console
+$ docker run -d -p 127.0.0.1:80:5000/udp training/webapp python app.py
 
 You also learned about the useful `docker port` shortcut which showed us the
 current port bindings. This is also useful for showing you specific port
 configurations. For example, if you've bound the container port to the
 `localhost` on the host machine, then the `docker port` output reflects that.
 
+```bash
     $ docker port nostalgic_morse 5000
 
     127.0.0.1:49155
+```
```

Comment:
These should use `console` (instead of `bash`), and the content inside the block should not be indented